### DISCUSSION TITLE
Timeshift backup

### DISCUSSION THREAD

**Discussion by arky217** [SpiralLinux/SpiralLinux-project#192] [Q&A]
I installed Spiral linux and didn't realize it made the partition btrfs instead of ext4.
After the installation, I used timeshift to backup the system on another drive, using the rsync type snapshot.
The backup seemed to go ok.
However, I thought it might be a good idea to create a btrfs partition on the backup drive and then
backup the system on that partition.
But it appears that you can only backup the btrfs type snapshot to a btrfs partition on the system drive only.

Question: Is my backup of the system btrfs partition using the rsync snapshot type on the backup drive
ext4 partition a valid backup.
In other words, if I needed to restore the backup, will it work ?

Or do I need to reinstall Spiral on a ext4 partition in order to use timeshift to back it up on a different drive ?

**Comment by artytux** [comment]
```
`@arky217` 
Question: Is my backup of the system btrfs partition using the rsync snapshot type on the backup drive
ext4 partition a valid backup.
In other words, if I needed to restore the backup, will it work ?
```

You know the only way to know 
Test the backup !

Regards
artytux
